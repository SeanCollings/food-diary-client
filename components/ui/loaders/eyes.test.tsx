import { render } from '@testing-library/react';
import { Eyes } from './';

describe('<Eyes>', () => {
  it('should render', () => {
    const { asFragment } = render(<Eyes />);
    expect(asFragment()).toMatchSnapshot();
  });
});
