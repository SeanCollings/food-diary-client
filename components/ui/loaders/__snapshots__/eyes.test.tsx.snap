// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Eyes> should render 1`] = `
<DocumentFragment>
  <span
    class="sc-66881844-0 bgOKGT loader"
    size="48"
  />
</DocumentFragment>
`;
